floris/simulation/grid.py

```diff
@@ -197,13 +197,16 @@ def set_grid(self) -> None:
             dtype=floris_float_type
         )
         # Calculate the radial distance from the center of the turbine rotor
-        disc_grid = np.linspace(
-            -1 * disc_area_radius,
-            disc_area_radius,
-            self.grid_resolution,
-            dtype=floris_float_type,
-            axis=1
-        )
+        if self.grid_resolution == 1:
```

Comment:
@bayc could you describe this change in a code comment?

Reply:
Added! I realized this was another bug if a grid resolution of 1 was selected, as the one point would be placed below the rotor center.